crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -111,8 +181,22 @@ def meta_title
 
     protected
 
+    def set_service_object#
+      @service_object = CrowbarService.new(logger)
+    end
+
+    def set_progess_values
+      @min_step = 1
+      @max_step = 10
+    end
+
     def hide_navigation
       @hide_navigation = true
     end
+
+    def repo_check
+      Openstack::Upgrade.check_ha_repo(logger) != false &&
+      Openstack::Upgrade.check_ceph_repo(logger) != false
```

Comment:
Style/MultilineOperationIndentation: Use 2 (not 0) spaces for indenting an expression spanning multiple lines.